python-lib/dku_utils/config_parser.py

```diff
@@ -0,0 +1,18 @@
+# Provide some utility methods to parse the saved configuration, clean it, normalize it and return in a predifined format (ex: command line args)
+
+SECURITY_GROUPS = 'securityGroups'
+SECURITY_GROUPS_ARG = '--node-security-groups'
+
+# retrieves the EKS Cluster SecurityGroups (network param),
```

Comment:
Python doc is not done this way. Its wrapped inside a block of `"""` and is indented just between the method name and the code.

Example of doc: https://github.com/dataiku/dip/blob/master/integration/python/scenario/visual_recipes/prepare/native_shaker/native_shaker_base.py#L348 (you can also specify the param/return type on the same line with another syntax)

Tips: 
* you can ask intellij or pycharm to help you a little : type `"""`  then enter on a new line after the method name.
* you can use CMD + J to generate a preview of the doc rendering to see if what you did make sense.